src/components/call/CallGeneralInfo.tsx

```diff
@@ -53,6 +74,62 @@ const CallGeneralInfo: React.FC<{
     }
   }, [startCall, endCall, formik]);
 
+  function validateRefnumber(input: string) {
+    let error;
+    const regExp = /((([a-z|\d]+)){)({digits:[1-9]+})/;
+    const prefix = /((([a-z|\d]+)){)/;
+    const suffix = /({digits:[1-9]+})/;
```

Comment:
Although those these aren't too complex, it might be helpful for future work to have a few comments explaining what these regexes are looking for :)